packages/bruno-app/src/providers/ReduxStore/slices/collections/index.js

```diff
@@ -1576,6 +1576,12 @@ export const collectionsSlice = createSlice({
           item.requestSent = action.payload.requestSent;
         }
 
+        if (type === 'request-skipped') {
+          const item = collection.runnerResult.items.find((i) => i.uid === request.uid);
+          item.status = 'skipped';
+          item.responseReceived = action.payload.responseReceived;
```

Comment:
What's the reason `item.responseReceived` is set here? 
Seems like a copy paste mistake but I could be wrong. 
I would expect `action.payload.responseReceived` to be undefined when the request is skipped so `item.responseReceived` would be as well?

Also why are we using `.find()` here, whereas all the other types use `.findLast()`?

Reply:
Hi,

You're right, ``responseReceived`` is no longer necessary.

Regarding the find vs findLast change, it was made after I checked out the branch, so I didn't catch it during the merge.

<img width="620" alt="Captura de ecrã 2024-08-27, às 20 06 51" src="https://github.com/user-attachments/assets/1826e040-72fb-4260-8cc8-f03ac4cde5d5">

Reply:
Thank you, looks good (on code level)

So have you been able to reproduce my error or do I need to set up some collection to reproduce the error?

Or did I use the function not as intended?